src/main/java/org/folio/rest/impl/TenantPermsAPI.java

```diff
@@ -106,8 +106,8 @@ public void postTenantpermissions(OkapiPermissionSet entity, Map<String, String>
                     if (dbPerm == null || Boolean.TRUE.equals(dbPerm.getDummy())) {
                       return Future.succeededFuture();
                     }
-                    if (dbPerm != null && Boolean.FALSE.equals(dbPerm.getMutable())
-                        && PermissionUtils.equals(perm, dbPerm)) {
+                    if (Boolean.TRUE.equals(dbPerm.getDeprecated()) || (Boolean.FALSE.equals(dbPerm.getMutable())
+                        && PermissionUtils.equals(perm, dbPerm))) {
```

Comment:
Can you please update the comment below?

Reply:
Sure.